pkgs/applications/networking/instant-messengers/mirage/default.nix

```diff
@@ -0,0 +1,30 @@
+{ mkDerivation, stdenv, lib, fetchFromGitHub, cmake, qtbase, qtquickcontrols2
+, qtkeychain, qtmultimedia, qttools, libqmatrixclient_0_5, libQuotient, gnome3
```

Comment:
libQuotient and libqmatrixclient are not used.

Reply:
Yeah, sorry, left in draft so people wouldn't review too closely. I just copied quaternion so there's lots of left over cruft. I'lll clean it up once it actually does something (anything) on launch.